.github/workflows/allo-allo.yml

```
@@ -40,10 +44,121 @@
+        if: "github.event.pull_request.merged == true"
+        runs-on: "ubuntu-latest"
+        steps:
+            - name: "Checkout repository"
+              uses: "actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683" # v4.2.2
+
+            - name: "Set Environment Variables"
+              env:
+                  AUTHOR: "${{ github.event.pull_request.user.login }}"
+                  REPO: "${{ github.event.repository.name }}"
+                  OWNER: "${{ github.event.repository.owner.login }}"
+              run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)